src/instructlab/lab.py

```diff
@@ -47,20 +46,22 @@
     show_default=True,
     help="Path to a configuration file.",
 )
-@click.version_option(package_name="instructlab")
 @click.pass_context
 # pylint: disable=redefined-outer-name
 def ilab(ctx, config_file):
     """CLI for interacting with InstructLab.
 
-    If this is your first time running InstructLab, it's best to start with `ilab init` to create the environment.
+    If this is your first time running ilab, it's best to start with `ilab init` to create the environment.
     """
-    # ilab init or "--help" have no config file. ilab sysinfo does not need one.
-    # CliRunner does fill ctx.invoke_subcommand in option callbacks. We have
-    # to validate config_file here.
     config.init_config(ctx, config_file)
 
 
+ilab.add_command(model_group.model)
+ilab.add_command(taxonomy_group.taxonomy)
+ilab.add_command(data_group.data)
+ilab.add_command(config_group.config)
```

Comment:
Was every commit prior to this one broken without these changes?

I don't think it's worth blocking on at this point, but it would be easier to follow all the changes with each of these included with the commit that created the group

Reply:
It looks like we're merging a bunch of commits that we know are in a broken state prior to this one. That's really not ideal, but I'm OK not blocking on it this time.